RiotSharpTest/RiotApiTest.cs

```diff
@@ -135,26 +135,22 @@ public void GetChampionAsync_Test()
 
         [TestMethod]
         [TestCategory("RiotApi")]
-        public void GetMasteryPages_Test()
+        public void GetMasteryPages_Summoner_HasMasteryPages()
```

Comment:
Can you change the name to GetSummoner_ExistingSummonerId_ReturnsMasteryPages()? I would be also nice to have a unit test for the case of the usage of an non existing summoner id.